app-header-layout/app-header-layout.html

```diff
@@ -147,6 +149,32 @@
         this._updateScroller();
       },
 
+      // Resizes app-header-layout to the height of the viewport and translates the content
+      // up to "simulate" disabled document scrolling on iOS.
+      simulateDisabledScroll: function(disabled) {
+        if (disabled) {
+          if (this.hasScrollingRegion) {
+            this.$.contentContainer.style.webkitOverflowScrolling = 'auto';
+          } else {
+            this._savedPageYOffset = window.pageYOffset;
+            this.style.height  = window.innerHeight + 'px';
```

Comment:
the new `iron-scroll-target-behavior` also defines a `_scrollTargetHeight`. Meanwhile, I could add that to `Polymer.AppDocumentScrollBehavior` so we don't need to write this code. No action needed yet.